### PR TITLE
WT-11016 Adjust the comment to remove outdated FIXME

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

File: src/btree/bt_cursor.c [hunk 1]
```diff
@@ -2161,10 +2161,10 @@ __wt_btcur_range_truncate(WT_CURSOR_BTREE *start, WT_CURSOR_BTREE *stop, WT_ITEM
     WT_STAT_DATA_INCR(session, cursor_truncate);
 
     /*
-     * FIXME WT-10887: Allow performing truncate operation without a timestamp on non logged tables
-     * for non standalone builds (MongoDB). MongoDB do perform truncate operation only on a table
-     * that do not have historical versions to avoid the problem. Remove this standalone build
-     * specific code when a proper solution is implemented.
+     * Allow performing truncate operation without a timestamp on non logged tables for non
+     * standalone builds (MongoDB). MongoDB do perform truncate operation only on a table that do
```
Comment (etienneptl) [inline src/btree/bt_cursor.c:2165]: `MongoDB do perform` -> `MongoDB does perform`

Comment (etienneptl) [inline src/btree/bt_cursor.c:2165]: `table that do` -> `table that does`

**Review by etienneptl — APPROVED** [review]
A few comments that don't need a second review. LGTM!

File: src/btree/bt_cursor.c (hunk 1 above)
Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.

Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.

File: src/btree/bt_cursor.c [hunk 2]
```diff
@@ -2161,10 +2161,10 @@ __wt_btcur_range_truncate(WT_CURSOR_BTREE *start, WT_CURSOR_BTREE *stop, WT_ITEM
...
+     * not have historical versions to avoid the problem. Remove this standalone build specific code
+     * when MongoDB switched to perform truncate operation with timestamps.
```
[the remark this answers is not in the document]
Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.
Regarding SERVER FIXME, not sure when the server is going to implement the switch to timestamp truncate. I am fine leave it without FIXME.

**Review by agorrod — APPROVED** [review]

**Pull request merged**